src/modules/Involvementapi.js

```diff
@@ -0,0 +1,24 @@
+const involvementAPI = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/';
+const involvementAPIid = 'Q1L23qalAk2Wt8OSXMKp';
+
+function getAllLikes() {
+  return fetch(`${involvementAPI}apps/${involvementAPIid}/likes/`)
+    .then((response) => response.json())
+    .then((data) => data);
```

Comment:
You are not handling errors in this request. Handling errors properly helps our app handle unexpected events gracefully, without confusing the user. We will add error handling later.